src/Configuration/ConfigurationFactory.php

```diff
@@ -90,7 +92,10 @@ public function create(
             $logVerbosity,
             $tmpDir,
             $schema->getPhpUnit(),
-            $schema->getMutators(),
+            (new MutatorParser(
+                $mutators,
+                (new MutatorsGenerator($schema->getMutators()))->generate()
+            ))->getMutators(),
```

Comment:
As I can see, we call `new MutatorsGenerator($schema->getMutators()))->generate()` twice for some reason.

Here, and inside [`MutatorParser::php`](https://github.com/infection/infection/blob/5dffa826a07b0f2ee6f6843aea621346f5194c05/src/Mutator/Util/MutatorParser.php#L80)

It's not an issue of this PR of course, but do you think this is something we should look at? 

From what I understand, we can first merge `$schema->getMutators()` and values from `--mutators`, and only then pass them to `(new MutatorsGenerator($mergedMutators))->generate()`

Reply:
That is a good point. I think I'm gonna focus on this mutators generation very soon; I would say right now it's not a performance issue big enough to be a blocker and that's how we were doing it before already as well.

So while I think we should go through, it's definitely something I/we should be aware of when touching that part again

Reply:
not a blocker at all, extracted to https://github.com/infection/infection/issues/844